src/xdist/dsession.py

```diff
@@ -457,3 +485,34 @@ def get_default_max_worker_restart(config):
         # if --max-worker-restart was not provided, use a reasonable default (#226)
         result = config.option.numprocesses * 4
     return result
+
+
+def get_workers_status_line(
+    status_and_items: Sequence[tuple[WorkerStatus, int]]
+) -> str:
+    """
+    Return the line to display during worker setup/collection based on the
+    status of the workers and number of tests collected for each.
+    """
+    statuses = [s for s, c in status_and_items]
+    total_workers = len(statuses)
+    workers_noun = "worker" if total_workers == 1 else "workers"
+    if status_and_items and all(s == WorkerStatus.CollectionDone for s in statuses):
+        first = status_and_items[0]
```

Comment:
I didn't understand at first why this only looks at the first worker. Would add a comment to clarify: "All workers collect the same items.". Also maybe write it as `tests_collected = status_and_items[0][1]`. Also maybe `tests_collected` -> `items_collected`, to match the terminology?

Reply:
Done!